crates/executors/src/executors/hooks/confirm.py

```diff
@@ -147,7 +147,10 @@ def main():
             json_success()
         elif status == "denied":
             reason = result.get("reason")
-            json_error(reason)
+            # Prefix user feedback with marker for extraction by Rust code
```

Comment:
Might be easier to pass the user feedback marker via the command line? Also, will be more robust to script version upgrades